torch/testing/_internal/common_utils.py

```diff
@@ -95,6 +95,10 @@
 from .composite_compliance import no_dispatch
 
 
+custom_device_mod = getattr(torch, torch._C._get_privateuse1_backend_name(), None)
+TEST_PRIVATEUSE1 = True if (hasattr(custom_device_mod, "is_available") and custom_device_mod.is_available()) else False
```

Comment:
can you move this definition to where all the other `TEST_{BACKEND}`s are defined ([referring to this part](https://github.com/pytorch/pytorch/blob/210b5d0546ea6b9571c4b14952492fe4c466ab3d/torch/testing/_internal/common_utils.py#L1066-L1067))

Reply:
> can you move this definition to where all the other `TEST_{BACKEND}`s are defined ([referring to this part](https://github.com/pytorch/pytorch/blob/210b5d0546ea6b9571c4b14952492fe4c466ab3d/torch/testing/_internal/common_utils.py#L1066-L1067))

Modified as requested